app/scripts/modules/core/src/pipeline/config/graph/PipelineGraphNode.tsx

```diff
@@ -168,7 +168,7 @@ export class PipelineGraphNode extends React.Component<IPipelineGraphNodeProps>
     );
 
     return (
-      <g onMouseEnter={this.highlight} onMouseLeave={this.removeHighlight} style={{ pointerEvents: 'bounding-box' }}>
+      <g onMouseEnter={this.highlight} onMouseLeave={this.removeHighlight} style={{ pointerEvents: 'all' }}>
```

Comment:
🤔❓

Reply:
Apparently `bounding-box` is no longer a valid option...